test/cmd-collect-delay.test.js

```diff
@@ -0,0 +1,47 @@
+const path = require('path')
+const { test } = require('tap')
+const rimraf = require('rimraf')
+const ClinicFlame = require('../index.js')
+
+test('cmd - test collect - 1s collect delay', (t) => {
+  const tool = new ClinicFlame({ debug: true, collectDelay: 1000 })
```

Comment:
We may find that the collectDelay needs to be a little bit higher, because `delayOneSecond` only starts running after all the Node.js bootstrapping is done, which might take a few dozen ms.

Reply:
@goto-bus-stop - Yeah, I arrived at a point on this where I was a bit stuck. Both delayOneSecond and delayTwoSecond were on the stack and I didn't know why.

Reply:
yeah, when running this branch of Flame with `--collect-delay 1000`, the little block on the right here is `delayOneSecond`, because we just barely measure a couple of ticks with that in it:

![image](https://user-images.githubusercontent.com/1006268/75242798-09a3e380-57c9-11ea-880e-0c24f8089c6e.png)

CI can be a bit unpredictable performance wise, so I would probably add something like 200-300ms to the `collectDelay` to be safe.

Reply:
@goto-bus-stop - I have now increased the delay to 1300ms.